Source/WebCore/crypto/keys/CryptoKeyEC.h

```diff
@@ -89,7 +92,18 @@ class CryptoKeyEC final : public CryptoKey {
     size_t keySizeInBytes() const { return std::ceil(keySizeInBits() / 8.); }
     NamedCurve namedCurve() const { return m_curve; }
     String namedCurveString() const;
+    
+#if OS(DARWIN) && !PLATFORM(GTK)
```

Comment:
Why do we need OS(DARWIN)?
Should we just have PLATFORM(COCOA)?

Reply:
I it is to handle that case in linux platforms